sklearnex/neighbors/common.py

```diff
@@ -264,3 +270,41 @@ def _onedal_gpu_supported(self, method_name, *data):
 
     def _onedal_cpu_supported(self, method_name, *data):
         return self._onedal_supported("cpu", method_name, *data)
+
+    def kneighbors_graph(self, X=None, n_neighbors=None, mode="connectivity"):
```

Comment:
Should X be optional here?

Reply:
would fail patching/ sklearn conformance tests otherwise, here is the sklearn source code https://github.com/scikit-learn/scikit-learn/blob/70fdc843a/sklearn/neighbors/_base.py#L949